contents/events/desir_winterschool2019/session01.mdx

```diff
@@ -0,0 +1,47 @@
+---
+number: 1
+title: Caring for data to shape the future
+speakers:
+  - mariafernanda-rollo
+#Add relevant names as listed in the person.yaml doc.
+synthesis: docs/1.1-Rollo.pdf
+---
+
+# Template for Event Sessions
+
+## Abstract
+
+_Keynote_
+
+Fernanda has a PhD in Contemporary History by the Faculdade de Ciências Sociais
+e Humanidades, Universidade Nova de Lisboa and is Associate Professor in the
+same faculty. Researcher and former President of the Institute for Contemporary
+History, Prof. Fernanda Rollo has coordinated research projects and has several
+texts published in the areas of Economic and Social History, History of
+Portugal, and history of innovation and organization of science. She is former
+Secretary of State for Science, Technology and Higher Education.
+
+<Speakers>
+
+<Speaker speaker="mariafernanda-rollo">
+
+Fernanda has a PhD in Contemporary History by the Faculdade de Ciências Sociais
+e Humanidades, Universidade Nova de Lisboa and is Associate Professor in the
+same faculty. Researcher and former President of the Institute for Contemporary
+History, Prof. Fernanda Rollo has coordinated research projects and has several
+texts published in the areas of Economic and Social History, History of
+Portugal, and history of innovation and organization of science. She is former
+Secretary of State for Science, Technology and Higher Education.
+
+<Youtube id="OqYJNUmhJzw" caption="Laurent Romary (Former DARIAH Director)" />
```

Comment:
is this youtube video here on purpose?

Reply:
It was just a test, videos should be ready soon.